modules/chapter2/pages/section3.adoc

```diff
@@ -0,0 +1,7 @@
+= Create a model
+
+This section covers how to create a Pytorch model and how to save it locally
```

Comment:
"locally" is somewhat ambiguous here (local laptop vs. local container). How about "This section covers how to create and save a PyTorch model"?

Reply:
Agreed. Done!